plugins/mongodb/mongostat.go

```diff
@@ -246,6 +247,8 @@ var StatHeaders = []StatHeader{
 	{"res", Always},
 	{"non-mapped", MMAPOnly | AllOnly},
 	{"faults", MMAPOnly},
+	{"lr|lw %", MMAPOnly | AllOnly},
```

Comment:
yea, best I can tell it's just some weird format on GH, original source -> https://github.com/mongodb/mongo-tools/blob/master/mongostat/stat_types.go#L265